tests/terraform/aws_s3_encryption_audit/.terraform.lock.hcl

```diff
@@ -0,0 +1,19 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
This file is a side-effect of 0.14+ terraform. It is a module lock file, like `package-lock.json` or `poetry.lock`. It's safe to have in repo, in general, but it means as the AWS provider moves forward we will need to explicitly upgrade these locks. Alternatively, we could just add this file pattern to .gitignore which means we get the same behavior from 0.13 terraform (fresh fetch of the providers each test run). The former is definitely more deterministic and could prevent breaks in tests from a future version of a terraform provider, but there will be overhead in having us periodically updating these locks.

Thoughts, @kapilt ?

Reply:
we should just add to .gitignore its a client side turd afaics reflecting local installation state.